Embedded/MaxMix/MaxMix.ino

```diff
@@ -266,23 +281,36 @@ bool ProcessPackage()
 
 
 //---------------------------------------------------------
+// \brief Encoder acceleration algorithm (Exponential - speed squared)
+// \param encoderDelta - step difference since last check
+// \param deltaTime - time difference since last check (ms)
+// \param volume - curent volume
+// \returns new adjusted volume
 //---------------------------------------------------------
-int8_t ComputeAcceleratedVolume(int8_t encoderDelta, uint32_t deltaTime, int8_t volume) {
+int8_t ComputeAcceleratedVolume(int8_t encoderDelta, uint32_t deltaTime, int16_t volume)
+{
   if (!encoderDelta)
     return volume;
 
-  if (deltaTime < ROTARY_ACCELERATION_SLOW_TIME) {
-    if (deltaTime < ROTARY_ACCELERATION_FAST_TIME) {
-      deltaTime = ROTARY_ACCELERATION_FAST_TIME;
-    }
+  float speed = (float)encoderDelta*1000/deltaTime;
+  float accelerationDivisor = max((1-(float)settings.accelerationPercentage/100)*400, 1);
```

Comment:
Don't make 400 a magic number, make it into a static const and add it to Config.h